### PR TITLE
BR: udpate BR auto tune

### PR DESCRIPTION
**Pull request by en-jin19**
### What is changed, added or deleted? (Required)
Update a part

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs-cn/pull/7515/files

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/br-auto-tune.md [hunk 1]
```diff
@@ -43,6 +43,10 @@ tikv-ctl modify-tikv-config -n backup.enable-auto-tune -v <true|false>
 
 - 问题 1：对于**以写负载为主的集群**，自动调节可能会让工作负载和备份进入一种“正反馈循环”：备份会占用较多资源，导致工作负载使用的资源变少。此时，自动调节会误以为资源使用率下降，从而让 BR 运行得更加激进。在这种情况下，自动调节实际上失效。
     - 解决方法：手动调节 `backup.num-threads`，限制处理备份的工作线程数量。
+        - 可以通过限制备份线程数量来限制备份对集群的影响的具体原因：
+
+          目前，备份过程会涉及大量的 SST 解码、编码、压缩、解压，而此过程会需要消耗大量的 CPU 资源。另外，以往的测试证明，在使用阻塞的 I/O 模型的场景下，单线程扫描过程和在内存中生成备份文件的过程可能会使用 100% 的 CPU。也就是说，即使不计算上传过程，备份任务实际上也会占用大量的 CPU 核心数。通过调整备份任务使用的线程数量，TiKV 可以控制备份任务使用的 CPU 核心数，从而减少其任务对集群性能的影响。
```
Comment (ran-huang) [inline br/br-auto-tune.md:48]:
```suggestion
    - 解决方法：手动调节 `backup.num-threads`，限制处理备份的工作线程数量。具体原理如下：

        目前，备份过程会涉及大量的 SST 解码、编码、压缩、解压，而此过程会需要消耗大量的 CPU 资源。另外，以往的测试证明，在使用阻塞的 I/O 模型的场景下，单线程扫描过程和在内存中生成备份文件的过程可能会使用 100% 的 CPU。也就是说，即使不计算上传过程，备份任务实际上也会占用大量的 CPU 核心数。通过调整备份任务使用的线程数量，TiKV 可以控制备份任务使用的 CPU 核心数，从而减少其任务对集群性能的影响。
```

Comment (YuJuncen) [reply br/br-auto-tune.md:48]: “在使用阻塞的 I/O 模型的场景下，单线程扫描过程和在内存中生成备份文件的过程可能会使用 100% 的 CPU。”
感觉可以换成⬇️
“备份过程中，备份线程池的 CPU 利用率接近 100%。”

Comment (YuJuncen) [reply br/br-auto-tune.md:48]: “也就是说，即使不计算上传过程，备份任务实际上也会占用大量的 CPU 核心数。”
感觉可以换成
“也就是说，备份任务会占用大量 CPU 资源。”
（感觉原版的说法似乎引入太多细节了🤔）

Comment (YuJuncen) [reply br/br-auto-tune.md:48]: 其他的部分我觉得没问题了～

Comment (en-jin19) [inline br/br-auto-tune.md]: updated, PTAL~ @ran-huang @YuJuncen

File: br/br-auto-tune.md [hunk 2]
```diff
@@ -42,7 +42,10 @@ tikv-ctl modify-tikv-config -n backup.enable-auto-tune -v <true|false>
 该功能的已知问题及其解决方案如下：
 
 - 问题 1：对于**以写负载为主的集群**，自动调节可能会让工作负载和备份进入一种“正反馈循环”：备份会占用较多资源，导致工作负载使用的资源变少。此时，自动调节会误以为资源使用率下降，从而让 BR 运行得更加激进。在这种情况下，自动调节实际上失效。
-    - 解决方法：手动调节 `backup.num-threads`，限制处理备份的工作线程数量。
+    - 解决方法：手动调节 `backup.num-threads`，限制处理备份的工作线程数量。具体原理如下：
+
+        目前，备份过程会涉及大量的 SST 解码、编码、压缩、解压，而此过程会需要消耗大量的 CPU 资源。另外，以往的测试证明，备份过程中，用于备份的线程池的 CPU 利用率接近 100%。也就是说，备份任务会占用大量 CPU 资源。通过调整备份任务使用的线程数量，TiKV 可以控制备份任务使用的 CPU 核心数，从而减少其任务对集群性能的影响。
```
Comment (en-jin19) [inline br/br-auto-tune.md:47]: updated, PTAL~ @ran-huang @YuJuncen

**Review by ran-huang — APPROVED** [review]

**Review by YuJuncen — APPROVED** [review]
lgtm

**Pull request merged**